src/modules/zk-evm-bridge/index.ts

```diff
@@ -5,3 +5,4 @@ export const stargateUrl = 'https://stargate.finance/transfer';
 export const relayBridgeAppLink = 'https://relay.link/bridge/astar-zkevm/?fromChainId=1';
 export const layerSwapLink =
   'https://www.layerswap.io/app?to=astarzk_mainnet&from=ethereum_mainnet';
+export const zkatanaBridgeUrl = 'https://bridge.gelato.network/bridge/zkatana';
```

Comment:
zKatana

Reply:
fixed in [this](https://github.com/AstarNetwork/astar-apps/pull/1238/commits/73807824eb5f82f09b41027309290152d704db13) commit

Reply:
That commit only capitalises the K in zKatana. 
That is not the commit you intended to submit I'm sure.

Reply:
I thought you asked just rename from 'zkatana' to 'zKatana'.